annotate.el

```diff
@@ -128,6 +130,11 @@ major mode is a member of this list (space separated entries)."
   :type  '(repeat symbol)
   :group 'annotate)
 
+(defcustom annotate-summary-link-max-width  64
```

Comment:
Why is it 64, instead of, say 72, 80, or `fill-column`, or `visual-wrap-column` (if set)?

Reply:
Because it is a power of 2! ;-)

Just kidding, but yes, it was totally arbitrary. I wonder if i could just use `window-body-width` and getting rid of this variable instead.

Reply:
Seems that `window-body-width` can be used! Is this solution ok for you?

Reply:
Absolutely! Thank you for investigating that!

Just out of curiosity, does the summary work correctly with crazy unicode shenanigans, like emoji or combining diacritics? It doesn't have to, if you ask me, but these kinds of things always were problematic for annotate.el.

Reply:
I would say is not a mess but there are rooms for improvements! :)